release/release.go

```diff
@@ -105,15 +103,13 @@ func readOSRelease() OS {
 	return osRelease
 }
 
-var ioutilReadFile = ioutil.ReadFile
+var fileExists = func(path string) bool {
+	_, err := os.Stat(path)
+	return err == nil
+}
```

Comment:
I've little experience in Go so a bit of help would be appreciated 😄. I think it'd be cleaner to use `osutils.FileExists`, but importing osutils creates a circular dependency:
```
./run-checks
Obtaining dependencies
Obtaining c-dependencies
HEAD is now at 74f4fe8 Merge pull request #63 from vasi/vasi-win-ci
Checking docs
Checking formatting
Running vet
package github.com/snapcore/snapd/osutil
        imports github.com/snapcore/snapd/release
        imports github.com/snapcore/snapd/dirs
        imports github.com/snapcore/snapd/dbusutil
        imports github.com/snapcore/snapd/testutil: import cycle not allowed in test

Crushing failure and despair.
```